apps/web/vibes/soul/form/checkbox-group/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { clsx } from 'clsx';
+import * as React from 'react';
+
+import { FieldError } from '@/vibes/soul/form/field-error';
+import { Label } from '@/vibes/soul/form/label';
+
+import { Checkbox } from '../checkbox';
+
+interface Option {
+  value: string;
+  label: string;
+  disabled?: boolean;
+}
+
+interface Props {
+  id?: string;
+  className?: string;
+  label?: string;
+  options: Option[];
+  errors?: string[];
+  name?: string;
+  value: string[];
+  onValueChange: (value: string[]) => void;
+}
+
+export function CheckboxGroup({
+  className,
+  label,
+  options,
+  errors,
+  name,
+  value,
+  onValueChange,
+}: Props) {
+  return (
+    <div className={clsx('space-y-2', className)}>
+      {label !== undefined && label !== '' && <Label>{label}</Label>}
+      <div className="space-y-2">
+        {options.map((option) => (
+          <Checkbox
+            aria-label={option.label}
+            checked={value.includes(option.value)}
+            key={option.value}
+            label={option.label}
```

Comment:
Seems weird to pass both `aria-label` and `label` here. Seems like we should only need one of these, not both

Reply:
Fixed.